src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -51,6 +62,12 @@ export const ConversationSpace: React.FunctionComponent = () => {
         </InformationCard>
       </InformationLeftColumn>
       <div className="conversation-space-container">
+        {!seatedAtTable && parentVenueId && parentVenue && (
+          <div className="back-map-btn" onClick={backToParentVenue}>
+            <div className="back-icon" />
+            <span className="back-link">Back</span>
+          </div>
+        )}
```

Comment:
Could you please reuse `BackButton` component?

Reply:
fixed in fae8270fb609124c5b2b41895cad7dced8bf0718